mithril-aggregator/src/file_uploaders/file_uploader.rs

```diff
@@ -0,0 +1,13 @@
+use async_trait::async_trait;
+use mithril_common::StdResult;
+use std::path::Path;
+
+pub type FileLocation = String;
```

Comment:
The name is a bit ambiguous, maybe this is better: `RemoteFileLocation`.
Also, an enum would be better here that an alias on a string.

Reply:
Suggestion:
`pub struct FileLocationUri(pub String);`

Reply:
Done ✅ , we've replaced it with `pub struct FileUri(pub String);`